app/views/hyrax/my/collections/_modal_add_collection_to_collection.html.erb

```diff
@@ -1,26 +1,27 @@
-<div class="modal fade" id="add-collection-to-collection-modal" tabindex="-1" role="dialog" aria-labelledby="add-collection-to-collection-label">
+<div class="modal fade" id="add-collection-to-collection-modal-<%= id %>" tabindex="-1" role="dialog" aria-labelledby="add-collection-to-collection-label">
+<% collection = Collection.find id %>
   <div class="modal-dialog" role="document">
     <div class="modal-content">
-      <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal" aria-label="<%= t('hyrax.dashboard.heading_actions.close') %>"><span aria-hidden="true">&times;</span></button>
-        <h2 class="modal-title" id="add-collection-to-collection-label"><%= t("hyrax.dashboard.my.action.add_collection_to_collection") %></h2>
-      </div>
-      <form class="add-collection-to-collection-form">
+      <%= form_tag(hyrax.dashboard_nest_collection_process_path(collection.id)) do %>
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="<%= t('hyrax.dashboard.heading_actions.close') %>"><span aria-hidden="true">&times;</span></button>
+          <h2 class="modal-title" id="add-collection-to-collection-label"><%= t("hyrax.dashboard.my.action.add_collection_to_collection") %></h2>
+        </div>
         <div class="modal-body">
-          <div class="form-group">
-            <label><%= t("hyrax.dashboard.my.action.select_collection") %></label>
-            <select class="form-control">
-              <option><%= t("hyrax.dashboard.my.action.select") %></option>
-              <option>Test collection hardcoded #1</option>
-              <option>Test collection hardcoded #2</option>
-            </select>
-          </div>
+          <label><%= t("hyrax.dashboard.my.action.select_collection") %></label>
+          <select name="parent_id">
+            <option value="none"><%= t("hyrax.dashboard.my.action.select") %></option>
+            <% colls = Hyrax::Collections::NestedCollectionQueryService.available_child_collections(parent: collection, scope: controller, limit_to_id: nil) %>
```

Comment:
We should evaluate whether NestedCollectionQueryService can be adjusted to work with the collection id instead of a collection instance.  We should try to avoid having to get the collection from Fedora if we can.